packages/block-library/src/cover/editor.scss

```diff
@@ -1,4 +1,6 @@
 .wp-block-cover {
+	position: relative;
```

Comment:
It looks like this rule was removed for some reason after the loading state was introduced for the Cover block (#25401).

Let's avoid unexpected regression and only apply this style conditionally. We can put `&.is-transient { position: relative; }` near other loading state classes (they start at L32).